src/saleor_app/app.py

```diff
@@ -12,18 +15,25 @@ class SaleorApp(FastAPI):
     def __init__(
         self,
         *,
+        manifest: Manifest,
         validate_domain: Callable[[DomainName], Awaitable[bool]],
         save_app_data: Callable[[DomainName, WebhookData], Awaitable],
         webhook_handlers: WebhookHandlers,
         get_webhook_details: Callable[[DomainName], Awaitable[WebhookData]],
+        use_insecure_saleor_http: bool = False,
+        development_auth_token: Optional[str] = None,
         **kwargs,
     ):
         super().__init__(**kwargs)
 
+        self.manifest = manifest
+        self.webhook_handlers = webhook_handlers
+        self.use_insecure_saleor_http = use_insecure_saleor_http
+        self.development_auth_token = development_auth_token
+
         self.extra["saleor"] = {
```

Comment:
the `FastAPI.extra` is not documented anywhere, it might be safer to get off this attribute.

Reply:
👍 
I think that there are some examples in fast API how to handle it better